client/src/pages/Home/Home.tsx

```diff
@@ -1,12 +1,64 @@
 import * as React from 'react';
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Stack,
+  Typography,
+} from '@mui/material';
+import { useNavigate } from 'react-router-dom';
 
 import { PrimaryDiv } from '../../assets/styles/styles';
 import Header from '../../components/Header';
+import { HomePageStack } from './styles';
+import { useUser } from '../../context/UserContext';
+import { homeInfo } from './constants';
 
 const Home = () => {
```

Comment:
I think we should add return types, and a JSDoc comment to this component definition